internal/cache/config.go

```diff
@@ -0,0 +1,33 @@
+package cache
+
+import (
+	application "github.com/debugger84/modulus-application"
+	"time"
+)
+
+type ModuleConfig struct {
+	MaxCacheSizeInMb int
+	CacheEnabled     *bool
+	LifeTime         time.Duration
+}
+
+func (s *ModuleConfig) InitConfig(config application.Config) error {
```

Comment:
Why don`t you use viper here?

Reply:
Forgot about changing the code in this place.